doc/benchmark/modules/ROOT/pages/configuration.adoc

```diff
@@ -104,6 +104,12 @@ Will log a lot of information.
 Use this only during development.
 Never use it during a load test as it slows down Gatling.
 
+| [.nowrap]`--http-proxy`
+| (not set)
+| If set, will proxy all HTTP requests to a proxy tool of your choice and the current default proxy host and port are `127.0.0.1`, `8888` respectively. These are currently not configurable.
+
+Never use it during a load test as it slows down Gatling to be able to proxy the requests to a proxy client and comes with the usual proxy nuances that we have to be careful, in general.
+
 | [.nowrap]`--sla-error-percentage`
```

Comment:
isn't `--close-http-connection` missing from the docs?

Reply:
I left it out given `--close-http-connection` is only implemented for the refresh token endpoint, similar to the other cli arguments `--refresh-token-period` or `--refresh-token-count` which were relevant only to a specific scenario, at least for now. But if you think, it would add value to expose it in the global config doc, I can add it here. lmk. @pruivo

Reply:
looks good to me :+1: thanks @kami619